codespell_lib/_codespell.py

```diff
@@ -413,8 +413,15 @@ def parse_options(args):
         for key in config['codespell']:
             # Add option as arg.
             cfg_args.append("--%s" % key)
+            # strip comments from lines
+            filtered_lines = []
+            for line in config['codespell'][key].splitlines():
+                line = line.strip()
+                if line != "":
+                    filtered_lines.append(line)
```

Comment:
```python
# strip whitespace from lines
filtered_lines = filter(None, line.strip() for line in config['codespell'][key].splitlines())
```
I don't understand `strip comments from lines`. This code does not strip comments, `configparser` does. This code strips whitespace.